src/relay/config/schema.py

```diff
@@ -28,7 +30,7 @@ class FeeSettingsSchema(Schema):
     base_fee = fields.Integer(missing=0)
     gas_price = fields.Integer(missing=0)
     fee_recipient = AddressField()
-    currency_network = AddressField(required=True)
+    currency_network = AddressField(missing=ZERO_ADDRESS)
```

Comment:
Please don't change that. If they miss to enter the address, they will be paid in the zero address currency network, i.e. not at all (which they may not have intended)